components/rules/rules.css

```diff
@@ -0,0 +1,35 @@
+.rules-main-container{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 4rem;
+}
```

Comment:
Just a suggestion, you can create a seperate class for flex & use it wherever required.

Reply:
Alright, I'll update it !